### PR TITLE
feat: add wallet details

### PR DESCRIPTION
**Pull request by Alex-A4**
## Description

copilot:all

## Type of Change
- Add TonWallet details page
- Add TokenWallet details page
- Add TokenWallet transactions widget

- [X] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: lib/app/router/routs/wallet/wallet.dart [hunk 1]
```diff
@@ -29,8 +33,36 @@ StatefulShellBranch get walletBranch {
             ),
           ),
           addSeedNamedRoute,
+          tonWalletDetails,
+          tokenWalletDetails,
```
Comment (nesquikm) [inline lib/app/router/routs/wallet/wallet.dart:37]: `Route` suffix would be expected here.

File: lib/feature/wallet/token_wallet_details/cubit/token_wallet_details_cubit.dart [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    account = a;
+    _cachedFiatBalance = currencyConvertService.convert(Fixed.zero);
+
+    final contract = assetsService.maybeGetTokenContract(
+      rootTokenContract,
+      nekotonRepository.currentTransport,
+    );
+    _cachedTokenBalance = Money.fromBigIntWithCurrency(
+      BigInt.zero,
+      Currency.create(contract?.symbol ?? '-', 0),
```
Comment (nesquikm) [inline lib/feature/wallet/token_wallet_details/cubit/token_wallet_details_cubit.dart:40]: Is it ok to use negative sign as currency symbol?

File: lib/app/router/routs/wallet/wallet.dart [hunk 3]
```diff
@@ -29,8 +33,36 @@ StatefulShellBranch get walletBranch {
             ),
           ),
           addSeedNamedRoute,
+          tonWalletDetails,
```
[the remark this answers is not in the document]
Comment (Alex-A4) [reply lib/app/router/routs/wallet/wallet.dart:36]: -

File: lib/app/router/routs/wallet/wallet.dart (hunk 1 above)
Comment (Alex-A4) [reply lib/app/router/routs/wallet/wallet.dart:37]: -

File: lib/feature/wallet/token_wallet_details/cubit/token_wallet_details_cubit.dart (hunk 2 above)
Comment (Alex-A4) [reply lib/feature/wallet/token_wallet_details/cubit/token_wallet_details_cubit.dart:40]: Replaced to `?` as told before

**Review by nesquikm — APPROVED** [review]

**Pull request merged**